### PR TITLE
Fix various typos + translate some comments to EN

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.ts,thirdparty,./toonz/sources/common/twain -L appy,ba,chunck,datas,forse,inbetween,inly,inout,pevent,possibile,strack,upto,vertexes`
Follow-up to PR #4156  
Closes #4072

### CODE REVIEW

File: toonz/sources/toonz/exportxsheetpdf.cpp [hunk 1]
```diff
@@ -1549,8 +1549,8 @@ void XSheetPDFTemplate::initializePage(QPdfWriter& writer) {
   QPageLayout pageLayout;
   pageLayout.setUnits(QPageLayout::Millimeter);
   pageLayout.setPageSize(
-      QPageSize(m_p.documentPageSize));  // ���ʂ�B4��ISO B4(250x353mm)
-                                         // ��{��B4��JIS B4(257x364mm)
+      QPageSize(m_p.documentPageSize));  // ���ʂ�B4��ISO B4(250x353mm)
+                                         // ���{��B4��JIS B4(257x364mm)
```
Comment (luzpaz) [inline toonz/sources/toonz/exportxsheetpdf.cpp:1553]: Sorry about this! I am so puzzled how these edits are making it in to my diffs. I guess it's something to do with Gitkraken, the gui git client I'm using. :angry: I'll try to revert this in vim

**Comment by luzpaz** [comment]
Sorry again for all the noise. Had to learn how to to an interactive staging via the CLI. Finally got rid of the rogue unicode change my git client was making. What a PITA :sweat_smile:    

That should be the last of the typos for now.

**Comment by luzpaz** [comment]
Spoke too soon :stuck_out_tongue_closed_eyes:
please also review 5d417bd and especially e1bfe76 (which may fix a bug?)

**Comment by RodneyBaker** [comment]
5d417bd is all good.  
e1bfe76 is interesting...

**Pull request merged**